core/src/test/java/nl/nn/adapterframework/pipes/SoapWrapperPipeTest.java

```diff
@@ -34,41 +41,119 @@ public void testUnwrap() throws Exception {
 		pipe.configure();
 		pipe.start();
 
-		String input = "<soapenv:Envelope xmlns:soapenv=\"http://schemas.xmlsoap.org/soap/envelope/\"><soapenv:Body>"
-				+"<root xmlns=\""+TARGET_NAMESPACE+"\">\n"
-				+"<attrib>1</attrib>\n"
-				+"<attrib>2</attrib>\n"
-				+"</root></soapenv:Body></soapenv:Envelope>";
-		String expected = "<root xmlns=\""+TARGET_NAMESPACE+"\">\n"
-				+"<attrib>1</attrib>\n"
-				+"<attrib>2</attrib>\n"
-				+"</root>";
+		String input = "<soapenv:Envelope xmlns:soapenv=\"" + SOAPConstants.URI_NS_SOAP_1_1_ENVELOPE + "\"><soapenv:Body>"
+				+ DEFAULT_BODY_END;
+		String expected = "<root xmlns=\"" + TARGET_NAMESPACE + "\">\n"
+				+ "<attrib>1</attrib>\n"
+				+ "<attrib>2</attrib>\n"
+				+ "</root>";
 
-		PipeRunResult prr = doPipe(pipe, input,new PipeLineSession());
+		PipeRunResult prr = doPipe(pipe, input, new PipeLineSession());
 
 		String actual = prr.getResult().asString();
 
 		TestAssertions.assertEqualsIgnoreCRLF(expected, actual);
 	}
 
+	@Test
+	public void testShouldKeepPipeSoapConfigurationVersionWhileWrapping() throws Exception {
+		pipe.setDirection(Direction.UNWRAP);
+		pipe.registerForward(new PipeForward("pipe2", "READY"));
+		pipe.setStoreResultInSessionKey("originalMessage_unwrapped");
```

Comment:
Dit doet als het goed is niets?

Reply:
Regel 62 bleek idd weg te kunnen. Thanks.